OOInteraction/src/commands/CSceneHandlerItemTest.cpp

```diff
@@ -26,7 +26,7 @@
 
 #include "CSceneHandlerItemTest.h"
 
-#include "VisualizationBase/src/items/RootItem.h"
+#include "VisualizationBase/src/Scene.h"
```

Comment:
You don't need this as it is already automatically included by some of the other includes here.